lib/mllp/client.ex

```diff
@@ -482,7 +490,21 @@ defmodule MLLP.Client do
         )
 
         error_reply = {:error, new_error(:sending, reason)}
-        {:keep_state_and_data, [{:reply, from, error_reply}]}
+
+        case data.reconnect_on_send_error do
+          true ->
+            data = stop_connection(data, reason)
+
```

Comment:
should we log a notice message here, so that we have some logs around it and it maybe good to have some visibility.

Reply:
Agreed, and done.